### PR TITLE
Feature/people empty states [PIL-459]

### PR DESCRIPTION
**Pull request by vyvil**
This PR adds Invite Friends banner to People screen (as empty state - when user does not have contacts yet or search provides 0 results) if referral feature is enabled.

### CODE REVIEW

File: src/screens/People/People.js [hunk 1]
```diff
@@ -399,7 +373,7 @@ class PeopleScreen extends React.Component<Props, State> {
     }, 1000);
   };
 
-  handleReferralBannerPress = () => {
+  handleInvitePress = () => {
     const { navigation, user } = this.props;
```
Comment (graszka22) [inline src/screens/People/People.js:377]: one nitpick: we have this code already in 3 different places and there is a high possibility we'll need it in another places, is there any possibility to move it to utils? or at least to the InviteBanner component?

File: src/screens/People/People.js [hunk 2]
```diff
@@ -409,6 +383,31 @@ class PeopleScreen extends React.Component<Props, State> {
...
+                bodyText={esBody}
+              />
+            </EmptyStateWrapper>
+          )
+          : (
+            <InviteBanner
+              title={inviteTitle}
+              onInvitePress={this.handleInvitePress}
+            />
+          )
```
Comment (poocart) [inline src/screens/People/People.js:405]: idk I would move this ternary handling before return and add the result var here instead, but as this piece is temporary (referrals going to be enabled anyway) – not sure if it makes sense 🤔

Comment (vyvil) [reply src/screens/People/People.js:405]: Yeah, didn't want to repeat Wrapper twice :)) and yeah, this flag will be removed eventually so there will be just one type of empty state :)

File: src/screens/People/People.js (hunk 1 above)
Comment (vyvil) [reply src/screens/People/People.js:377]: Hm, this could be made into an action. I'll take a look :)

Comment (vyvil) [reply src/screens/People/People.js]: Done :)

**Pull request merged**